src/courseware/course/sidebar/common/Sidebar.test.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Factory } from 'rosie';
+import {
```

Comment:
[code style]: Let’s separate external library imports from local imports with a blank line for better readability.

```suggestion
import { Factory } from 'rosie';

import {
```

Reply:
Should the line `import { createRef } from 'react';` come before `import { Factory } from 'rosie';`? The idea is to separate library imports from local imports.

Reply:
fixed